indico/MaKaC/webinterface/rh/registrationFormModif.py

```diff
@@ -228,8 +229,8 @@ def map(self, params):
         if params.get("addvalue","").strip()!="":
             sv=StatusValue(self)
             cap=params.get("newvalue")
-            if cap.strip()!="":
-                sv.setCaption(params.get("newvalue"))
+            if cap.strip() and cap not in map(lambda val: val.getCaption(), self._values):
```

Comment:
once again, don't pass a lambda to map

`not any(cap == val.getCaption() for val in self._values)` is much more efficient here. another option would be `cap not in {val.getCaption() for val in self._values}` but why build a set when you don't really need it